app/views/carts/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="container text-center">
+  <div class="row">
+    <div class="col-md-2 mb-3 cart-info">
+      <i class="fas fa-shopping-bag shopping-bag"></i>
+      <p class="cart-title-counter">items:<%= @cart.line_items.count %></p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-1 mb-4 ">
+      <div class="tooltip-custom">
+        <%= button_to 'Clear Cart', cart_clear_cart_path(cart_id: @cart.id), method: :delete, class: 'btn btn-danger remove-all-button' %>
```

Comment:
`cart_clear_cart_path` maybe a better naming convention for route and its helpers